app/build.gradle

```diff
@@ -11,8 +12,8 @@ android {
         applicationId "chat.rocket.android"
         minSdkVersion 21
         targetSdkVersion versions.targetSdk
-        versionCode 1001
-        versionName "2.0.0-dev1"
+        versionCode 1005
```

Comment:
What's the criteria used to update the versionCode?

Reply:
For now, each version sent get a versionCode bump for update the apk on the smartphone